### PR TITLE
Add a script to perform load test to create and search records

### PR DESCRIPTION
**Pull request by gimite**
#180

### CODE REVIEW

File: tests/load_test/generate_names.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+"""A script to generate names_in_db.txt and names_not_in_db.txt, name lists
+used in load_test.py.
+
+Usage:
+  $ python tests/load_test/generate_names.py
+
+It generates mixture of English and Japanese names. Japanese names are added
+because Person Finder performs some Japanese specific handling in search.
+
+English names are generated using random 3-10 Latin alphabets. This is because
```
Comment (kevinalle) [inline tests/load_test/generate_names.py]: random 3-10 Latin alphabet characters

File: tests/load_test/generate_names.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+LATIN_ALPHABETS = 'abcdefghijklmnopqrstuvwxyz'
+
+
+def generate_english_name_component():
+    name_len = random.randint(ENGLISH_NAME_MIN_LENGTH, ENGLISH_NAME_MAX_LENGTH)
+    return ''.join(random.choice(LATIN_ALPHABETS) for _ in xrange(name_len))
+
+
+if __name__ == '__main__':
+	# To get reproducible output.
```
Comment (kevinalle) [inline tests/load_test/generate_names.py]: indent

File: tests/load_test/load_test.py [hunk 3]
```diff
@@ -0,0 +1,418 @@
...
+      "num_queries": 100,
+      "create_record_qps": 10,
+      "search_record_qps": 10,
+      "num_threads": 100,
+      "output_dir": "/tmp/load_test_result",
+      "test_mode": false
+    }
+
+  - Run this command to perform load test to create person records:
+    $ ./tools/load_test config.json create
```
Comment (kevinalle) [inline tests/load_test/load_test.py]: ./tools/load_test.py?

File: tests/scrape.py [hunk 4]
```diff
@@ -296,7 +296,7 @@ class Session:
         message - the status message of the last request
         headers - the headers of the last request as a dictionary
         content - the content of the last fetched document
-        doc     - the Region spanning the last fetched document
+        doc     - the Document currently opened.
```
Comment (kevinalle) [inline tests/scrape.py]: doc     - the document currently opened

**Comment by kevinalle** [comment]
LGTM! just nits

File: tests/load_test/generate_names.py (hunk 1 above)
Comment (gimite) [inline tests/load_test/generate_names.py]: Done.

File: tests/load_test/generate_names.py (hunk 2 above)
Comment (gimite) [inline tests/load_test/generate_names.py]: Done.

File: tests/scrape.py (hunk 4 above)
Comment (gimite) [inline tests/scrape.py]: I meant an instance of a class "Document". I rephrased to "Document instance".

**Comment by gimite** [comment]
Thanks, merged after applying your comments + one more fix needed to make the server test work.

**Pull request merged**